org.openshot.OpenShot.json

```diff
@@ -0,0 +1,385 @@
+                }
+            ]
+        },
+        {
+            "name": "chardet",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=/app --root=/"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://pypi.python.org/packages/fc/bb/a5768c230f9ddb03acc9ef3f0d4a3cf93462473795d18e9535498c8f929d/chardet-3.0.4.tar.gz",
+                    "sha256": "84ab92ed1c4d4f16916e05906b6b75a6c0fb5db821cc65e70cbd64a3e2a5eaae"
+                }
+            ]
+        },
+        {
+            "name": "certifi",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=/app --root=/"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://pypi.python.org/packages/23/3f/8be01c50ed24a4bd6b8da799839066ce0288f66f5e11f0367323467f0cbc/certifi-2017.11.5.tar.gz",
+                    "sha256": "5ec74291ca1136b40f0379e1128ff80e866597e4e2c1e755739a913bbc3613c0"
+                }
+            ]
+        },
+        {
+            "name": "idna",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=/app --root=/"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://pypi.python.org/packages/f4/bd/0467d62790828c23c47fc1dfa1b1f052b24efdf5290f071c7a91d0d82fd3/idna-2.6.tar.gz",
+                    "sha256": "2c6a5de3089009e3da7c5dde64a141dbc8551d5b7f6cf4ed7c2568d0cc520a8f"
+                }
+            ]
+        },
+        {
+            "name": "openshot-qt",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=/app --root=/"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/OpenShot/openshot-qt/archive/v2.4.1.tar.gz",
+                    "sha256": "da388c08cd94ec503a57756871244d9bb8746973807e2f95618a0d9d5e11fae9"
+                },
+                {
+                    "type": "file",
+                    "path": "org.openshot.OpenShot.appdata.xml"
+                }
+            ],
+            "post-install": [
+                "mv /app/share/applications/openshot-qt.desktop /app/share/applications/org.openshot.OpenShot.desktop",
+                "desktop-file-edit --set-key=Icon --set-value='org.openshot.OpenShot' /app/share/applications/org.openshot.OpenShot.desktop",
+                "mv /app/share/mime/packages/openshot-qt.xml /app/share/mime/packages/org.openshot.OpenShot.xml",
+                "mkdir -p /app/share/icons/hicolor/scalable/apps",
```

Comment:
Might as well use `install` rather than `mkdir` and `mv`.

Reply:
But wouldn't that copy the file?

Reply:
Oh sorry yes, I was thinking `cp` for some reason.